src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -1835,14 +1841,15 @@ void EmitCapture(RegexNode node, RegexNode? subsequent = null)
                 }
             }
 
-            // Emits the code to handle a positive lookahead assertion.
-            void EmitPositiveLookaheadAssertion(RegexNode node)
+            // Emits the code to handle a positive lookaround assertion. This is a positive lookahead
+            // for left-to-right and a positive lookbehind for right-to-left.
+            void EmitPositiveLookaroundAssertion(RegexNode node)
```

Comment:
NIT: Isn't this name a bit misleading though? PositiveLookaround should in theory mean both positive lookahead or positive lookbehind for both left-to-right and right-to-left, yet from the description above means this is really always a positive lookahead except that in right-to-left the direction is opposite. Anyway, small detail which probably doesn't matter.

Reply:
> PositiveLookaround should in theory mean both positive lookahead or positive lookbehind for both left-to-right and right-to-left

It does. The node's RegexOptions defines the direction, so it's lookahead if (node.Options & RegexOptions.RightToLeft) == 0 and it's lookbehind if (node.Options & RegexOptions.RightToLeft) != 0.